domovoi.py

```diff
@@ -60,7 +60,7 @@ def run(self, processes_file):
         # Watch over the processes and respond accordingly
         while True:
             for solar_car_process in solar_car_processes:
-                if solar_car_process.check_status():
+                if solar_car_process.check_status() is not None:
```

Comment:
It is rare use `is`  in python, in comparison to the `==` operator, `is` checks if they are literally the exact same object at the same address space. Be in a habit of using `==` over `is`, in this use case `==` is much more appropriate.